robotoff/app/core.py

```diff
@@ -78,24 +99,34 @@ def get_insights(
 
     if order_by is not None:
         if order_by == "random":
-            query = query.order_by(peewee.fn.Random())
+            query = query.order_by((peewee.fn.Random() * ProductInsight.n_votes).desc())
```

Comment:
I'm curious if this is effective enough on a large number of insight, to make voted one really get first places ? (I can see it multiplies expected value, but I have the feeling that on a large collection, we won't get so much voted results !)

Reply:
If I'm not mistaken, insights with n+1 votes will be (n+1)/(n) more likely to bubble to the top than insights with n votes. It's not perfect, but it doesn't need to be. I suggest we wait and see how it ends up working?

Reply:
yep, it's just that if you have 1000 insight, and 10 with 2 votes, you will only have  20 / 1010 chances of getting an already voted one. But You're right that this has the advantage of being very simple :-)
Otherwise I was thinking something like:
random / ((4 - votes) * alpha)
where alpha could be 4